resources/CICD/GithubActions/test.yml

```diff
@@ -0,0 +1,56 @@
+name: test
+
+on: 
+  push:
+    branches:  
+      - '!releases/**'
+      - '!develop'
+      - '!master'
```

Comment:
I would prefer to use `pull request`, we just need to run the CI for the pull request. If we run for all working branches, it will take a lot of resource.